app/templates/macros/toolkit_forms.html

```diff
@@ -1,3 +1,14 @@
+{#
+The following form macros extend on top of of the macros found in the front end toolkit so we can support additional
+functionality needed for forms for applying for a framework.
+
+So we can parse question references for the supplier framework declaration forms, we redefine all our individual
```

Comment:
It definitely feels like this is something that doesn't really belong in the templating layer. But I don't really quite understand well enough what it's doing - why do we parse the question when displaying it? If there's _some_ further clarification of what's going on, that we could put in the comments here, then that would be nice.

Reply:
Good point, it feels like this logic should be done by the content loader in preparing the content ready to be passed to the template. Not too sure how hard it would be to move the logic into the content loader but may not be a simple job.

I'll have a quick think but I'll probably not try and move it in as it doesn't feel like a simple job (and if I don't then I'll add a tech debt ticket about it). However I will try and fill out some more details in the documentation here.

Reply:
Will rebase on top of this with some even better comments :)